docs/pages/en-us/Web-Service.md

```diff
@@ -0,0 +1,263 @@
+
+## Audit Web Service Client
+
+The Audit Library can be used in your audited Java application for building and sending audit event messages to the Audit Web Service.
+
+### Generating a Client-side Auditing Library
+
+The Getting Started guide's [Writing an Application Audit Event Definition File](Getting-Started#writing-an-application-audit-event-definition-file) and [Generating a Client-side Auditing Library](Getting-Started#generating-a-client-side-auditing-library) sections cover the generation of a type-safe client-side auditing library. Generating a client-side auditing library offers a type-safe alternative to invoking the auditing library's event builder or the Audit Web Service REST API.
+
+### Using the Audit Web Service Client
+
+Once you have your auditing library (generated or `caf-audit`), you use it to send audit events to the Audit Web Service.
+
+#### Dependencies
+
+If you generated a client-side library it should be referenced in the normal way in the application's POM file. You shouldn't need to manually add a dependency on `caf-audit` as it will be a transitive dependency of the generated library.
+
+#### Audit Connection
+
+Regardless of whether you choose to use a generated client-side library, or to use `caf-audit` directly, you must first create an `AuditConnection` object.
+
+This object represents a logical connection to an endpoint which, in this case, is the Audit Web Service. It is a thread-safe object. ***Please take into account that this object requires some time to construct. The application should hold on to it and re-use it, rather than constantly re-construct it.***
+
+The `AuditWebServiceClientConnection` object, which is the `AuditConnection` object implementation for creating a connection to the Audit Web Service, can be constructed using the static `createConnection()` method in the `AuditConnectionFactory` class. This method takes a `ConfigurationSource` parameter, which is the standard method of configuration in CAF. To enable the creation of the `AuditWebServiceClientConnection` object, with the `AuditConnectionFactory`, the `CAF_AUDIT_MODE` environment variable must be set to `webservice`.
+
+##### ConfigurationSource
+
+[comment]: <> (The caf-audit Client-API.md documentation content contains duplication of the ConfigurationSource section. It is important that any changes here must also be included within the Client-API.md content.)
+
+You may already have a CAF configuration source in your application. It is a general framework that abstracts the source of the configuration, allowing it to come from any of the following:
+
+* environment variables
+* files
+* a REST service
+* a custom source that better integrates with the host application.
+
+If you're not already using CAF's configuration mechanism, this sample code illustrates the generation of a ConfigurationSource object.
+
```

Comment:
I don't find the below example helpful.  In fact I find it very confusing as I can't see how it relates to the Audit Service in any way.

Would something like the following not be better?

```
    private static ConfigurationSource getWebServiceToElasticConfig()
    {
        return new ConfigurationSource() {
            @SuppressWarnings("unchecked")
            @Override
            public <T> T getConfiguration(Class<T> type) throws ConfigurationException
            {
                WebServiceClientAuditConfiguration config = new WebServiceClientAuditConfiguration();
                config.setWebServiceEndpoint("http://" + monkeyConfig.getWsHostnameAndPort() + "/caf-audit-service/v1");
                return (T) config;
            }
        };
    }
```

@mulhollandc please on comment on this also